README.md

```diff
@@ -15,8 +15,22 @@ make test-kafka-broker-upstream-nightly
 # artifacts in installation/manifests/product-nightly
 make test-kafka-broker-midstream-nightly
 
+# Deploy requirements (Kafka, Serverless, Hyperfoil) from products and run a specific test
+TEST_CASE=tests/broker/kafka/p10-r3-ord-b10-t10-64kb ./bin/run_midstream_nightly.sh
+
 # Run a specific test using an existing Serverless or Knative installation
-TEST_CASE=tests/broker/kafka/p10-r3-ordered ./bin/run_test.sh
+TEST_CASE=tests/broker/kafka/p10-r3-ord-b10-t10-64kb ./bin/run_test.sh
+
+# Same as above, but skip deleting resources after the test for debugging purposes
+export SKIP_DELETE_RESOURCES=true
+TEST_CASE=tests/broker/kafka/p10-r3-ord-b10-t10-64kb ./bin/run_test.sh
+
+# Run quick smoke test if requirements are already installed
```

Comment:
would using https://github.com/openshift-knative/eventing-hyperfoil-benchmark/blob/main/bin/run.sh work for this case?

Reply:
Sorry I didn't understand.

```
# Run quick smoke test if requirements are already installed
```

You mean for this case?

In that case, I would want to avoid using `run.sh` as it applies manifest and stuff. I want to skip that part as in this case the assumption is that the requirements are installed.

Reply:
It calls `apply_manifests` and that does more than that, including machineset configs and etc.: https://github.com/aliok/eventing-hyperfoil-benchmark/blob/08d11e2322fde0ed8c163b59705c87f231161846/bin/common.sh#L55

Reply:
I see, that's bad